app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      session[:user_id]= user.id
+      redirect_to '/dashboard'
+    else
+      flash[:register_errors] = user.errors.full_messages
+      redirect_to '/'
+    end
+  end
+
+  def show
+    event = Event.all
```

Comment:
If you want to use the `event` variable in the show view you need to use an instance variable, see #5.